src/app/public/modules/type-definitions/directive-property-definition.ts

```diff
@@ -1,19 +0,0 @@
-import {
-  SkyDocsPropertyDecorator
-} from './property-decorator';
-
-import {
-  SkyDocsPropertyDefinition
-} from './property-definition';
-
-export interface SkyDocsDirectivePropertyDefinition extends SkyDocsPropertyDefinition {
```

Comment:
This type is redundant because it's exactly the same as `SkyDocsPropertyDefinition`.